code/modules/mob/living/simple_animal/shade.dm

```diff
@@ -26,21 +26,32 @@
 	faction = list("cult")
 	status_flags = CANPUSH
 	flying = 1
+	loot = list(/obj/item/weapon/ectoplasm)
+	del_on_death = 1
 
 
 /mob/living/simple_animal/shade/death()
-	..(1)
-	new /obj/item/weapon/ectoplasm (src.loc)
-	visible_message("<span class='warning'>[src] lets out a contented sigh as their form unwinds.</span>")
-	ghostize()
-	qdel(src)
-	return
+	deathmessage = "<span class='warning'>[src] lets out a contented sigh as their form unwinds.</span>"
+	..()
 
+/mob/living/simple_animal/shade/Process_Spacemove(movement_dir = 0)
+	return 1

```

Comment:
All the other ones aren't but probably, yes. It's one of the things under "someone else's problem"